bedrock/firefox/templates/firefox/browsers/chromebook.html

```diff
@@ -36,21 +36,21 @@ <h2 class="c-sub-navigation-title">{{ ftl('sub-navigation-firefox-for-desktop',
 {% endblock %}
 
 {% block content %}
-<div class="mzp-l-content">
-  {% call hero(
-    title=ftl('browsers-chromebook-get-firefox-browser'),
-    class='mzp-has-image mzp-t-firefox mzp-t-product-firefox',
+  {% call split(
     image_url='img/firefox/browsers/quantum/browser.jpg',
+    include_highres_image=True,
     heading_level=1
   ) %}
+  <div class="mzp-c-logo mzp-t-logo-lg mzp-t-product-firefox"></div>
+  <h1 class="mzp-u-title-xl">{{ ftl('browsers-chromebook-get-firefox-browser') }}</h1>
   <p>{{ ftl('browsers-chromebook-so-youve-got') }}</p>
 
   <div class="mzp-c-menu-list mzp-t-cta mzp-t-download" id="chromebook-download-list">
     <h4 class="mzp-c-menu-list-title">{{ ftl('browsers-chromebook-dropdown-copy') }}</h4>
     <ul class="mzp-c-menu-list-list download-platform-list">
       <li class="mzp-c-menu-list-item">
         <a href="{{ firefox_adjust_url('android', 'firefox-browsers-faq') }}" rel="external noopener" data-link-type="download" data-display-name="Android" data-download-version="android" data-download-os="Android">
-          {{ ftl('browsers-chromebook-get-firefox-for') }}
+          {{ ftl('browsers-chromebook-browsers-chromebook-get-firefox-for') }}
```

Comment:
This is actually a bug fix. The canonical string ID has the repeated bit https://github.com/mozilla/bedrock/blob/master/l10n/en/firefox/browsers/chromebook.ftl#L13

But there also seems to be a bug in the current template because the tagline and dropdown don't appear at all on the current page (see prod https://www.mozilla.org/firefox/browsers/chromebook/), I expect due to some problem with how the hero macro is called but I didn't investigate deeply. When I replaced the hero I fixed that glitch and the menu appeared and I discovered the mismatched string ID. We can fix the ID as well but that makes it a new string and this has already been widely translated.

Reply:
Bug fix is fine as is 👍